test/powershell/Language/LanguageTestSupport.psm1

```diff
@@ -76,7 +115,7 @@ function ShouldBeParseError
         if ($SkipAndCheckRuntimeError) 
         {
             It "error should happen at parse time, not at runtime" -Skip {}
-            $errors = Get-RuntimeError -Src $src
+            $errors = Get-RuntimeError -Src $src -Timeout 5

```

Comment:
I'd feel a bit better if we waited a bit longer, maybe 10 seconds - in case we're on a really slow VM or we add a test that is somewhat longish (like 1s or so).